keys/crypto.go

```diff
@@ -150,18 +150,18 @@ func newP256Keypair() (*ecdsa.PublicKey, *ecdsa.PrivateKey, error) {
 	return privkey.Public().(*ecdsa.PublicKey), privkey, nil
 }
 
-// newNebulaX25519KeypairPEM returns a new Nebula keypair (X25519) in PEM format.
+// newNebulaX25519KeypairPEM returns a new Nebula key-agreement keypair (ED25519) in PEM format.
```

Comment:
suuuuuper nit, but i think it's Ed25519. i don't think the D stands for anything.

what does key-agreement mean in this context exactly? what other types of Nebula keypairs exist?

Reply:
the public keys we use for CAs are "signing keys", not "key agreement keys". You're _technically_ not allowed to interchange them.